### ISSUE TITLE
Can't dye Bloodlord belt

### ISSUE THREAD

**Issue by ElMosquito** [WarEmu/WarBugs#16184]
With the recent changes to gear appearance I've noticed that the Bloodlord belt for DoK can't be dyed. I used to have that appearance on my DoK before the changes and would like to keep using it but a red belt doesn't fit with the rest gear being dyed blue.

**Comment by Kaelang** [comment]
Was you able to dye the item before the appearance change?

This being said, taken from another report - some items simply cannot be dyed as they are coded in such a way from live and the functionality to change this isn't available right now. 

I'll keep this open awaiting further information from yourself for now.

**Comment by ElMosquito** [comment]
Yes, the item is dyed and it can still be dyed but it doesn't show as appearance. Unfortunately I don't have any good screenshots from before but I have a screenshot on the belt showing it being dyed according to the text but the appearance is still in original colors.
![Bloodlordbelt](https://user-images.githubusercontent.com/16590792/79862253-1f531500-83d6-11ea-9e4b-c8bf9f9d489a.jpg)

**Comment by Kaelang** [comment]
Could you delete your cache and try a relog to see if this is only visible from your perspective?

**Comment by ElMosquito** [comment]
I deleted my cache and it still doesn't show any dyes.
